roles/tas_single_node/tasks/main.yml

```
@@ -8,3 +8,6 @@
 
 - name: Execute Podman Actions
   ansible.builtin.include_tasks: podman.yml
+
+- name: Check OS System Support
```

We should execute these tasks at the very start, so that we fail fast and users don't observe random failures because things don't work on their system.

---

ah yes, that makes sense! Thanks :+1: I will adress it :)